tuf-spec.md

```diff
@@ -1057,16 +1069,20 @@ used to download the timestamp metadata file is of the fixed form FILENAME.EXT
 
   * **2.1**. **Check signatures.** The new timestamp metadata file must have
   been signed by a threshold of keys specified in the trusted root metadata
-  file.
+  file.  If the new timestamp metadata file is not properly signed, discard it
+  and report the signature failure.
```

Comment:
Is it clear enough in this addition that the update cycle should be aborted?

Comment holds for the modifications above and below, too, though that can be clarified once at the top instead (see first review comment).